clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -865,6 +886,8 @@ public void onAssignment(Set<TopicPartition> assignment) {
         private final TopicPartition partition;
         private final CompletedFetch completedFetch;
         private final Iterator<? extends RecordBatch> batches;
+        private final Set<Long> abortedPids;
+        private final Queue<FetchResponse.AbortedTransaction> abortedTransactions;
```

Comment:
I think your comment is for the priority queue used in the producer. All I was suggesting here is that we replace `Queue` with `PriorityQueue` to emphasize that not just any queue will work.